tests/explorer.api.tests/QueryTests.cs

```diff
@@ -169,6 +170,37 @@ public async void TestRepeatingRows()
             });
         }
 
+        [Fact]
+        public async void TestCancelQuery()
+        {
+            using var client = new HttpClient() { BaseAddress = TestWebAppFactory.Config.AircloakApiUrl };
```

Comment:
You could use the VCR here. 

```suggestion
            var vcrCassetteInfo = factory.GetVcrCasetteInfo(nameof(QueryTests), nameof(TestCancelQuery));
            using var client = factory.CreateAircloakApiHttpClient(vcrCassetteInfo);
```

Reply:
fixed with 1291bc2d3824a5c765573950679d8817613a7754